python/paddle/fluid/layers/nn.py

```diff
@@ -326,6 +327,8 @@ def embedding(input,
     """
 
     helper = LayerHelper('embedding', **locals())
+    if remote_prefetch:
```

Comment:
remote_prefetch and is_distributed are confusing, any way to remove one?

Reply:
remote_prefetch is deleted and will be replaced by is_sparse in the future.